src/app/creative-team/dashboard/tables/tables.component.html

```diff
@@ -0,0 +1,90 @@
+<div fxLayout="row" fxLayoutAlign="start center" fxLayoutGap="10px">
+  <mat-card fxFlex="50" class="main-card">
+    <mat-card-header class="main-header">
+      <mat-card fxFlex="100" class="header" fxLayout="row" fxLayoutAlign="start center"
+                [ngStyle]="{background:'#BC6EC9'}">
+        <mat-nav-list fxLayout="row" fxLayoutAlign="start center" fxLayoutGap="10px"[ngStyle]="{color:'#FFFFFF'}">
+          <span>Tasks:</span>
+          <a mat-list-item routerLinkActive="list-item-active" routerLink="/" [ngStyle]="{color:'#FFFFFF'}">
+            <mat-icon>bug_report</mat-icon>
+            BUGS</a>
+          <a mat-list-item routerLinkActive="list-item-active" routerLink="/" [ngStyle]="{color:'#FFFFFF'}">
+            <mat-icon>code</mat-icon>
+            WEBSITE</a>
+          <a mat-list-item routerLinkActive="list-item-active " routerLink="/" [ngStyle]="{color:'#FFFFFF'}">
+            <mat-icon>cloud</mat-icon>
+            SERVER</a>
+        </mat-nav-list>
+      </mat-card>
+    </mat-card-header>
+    <br>
+    <br>
+    <br>
+    <!--    content-->
+    <mat-card-content class="main-content">
+      <div *ngFor="let word of words">
+        <div fxLayout="row" fxLayoutGap="30px" fxLayoutAlign="space-between center" class="table">
+          <mat-checkbox class="example-margin">{{word.info}}</mat-checkbox>
+          <div fxLayout="row" fxLayoutGap="10px" fxLayoutAlign="start center">
+            <mat-icon
+              matTooltip="Edit  Task"
+              [matTooltipPosition]="position.value!"
+              aria-label="Button that displays a tooltip when focused or hovered over" [ngStyle]="{'color':word.edit}">edit
```

Comment:
aria-label="Button that displays a tooltip when focused or hovered over" Is it necessary to use this